api/specs/webserver/openapi-tasks.yaml

```diff
@@ -0,0 +1,70 @@
+paths:
+  /tasks:
+    get:
+      operationId: list_tasks
+      tags:
+        - tasks
+      responses:
+        "200":
+          description: Lists all the tasks currently available/running
```

Comment:
I would have preferred if this was filtered by user_id.

Reply:
Question: where was this implemented, I did not see it. Was it a previous pull request?